data/json/items/armor/boots.json

```diff
@@ -2703,6 +2703,20 @@
     "flags": [ "VARSIZE", "SKINTIGHT" ],
     "armor": [ { "coverage": 100, "covers": [ "foot_l", "foot_r" ] } ]
   },
+  {
+    "id": "toesocks",
+    "type": "ARMOR",
+    "name": { "str": "pair of toe socks", "str_pl": "pairs of toe socks" },
+    "description": "A pair of comfortable socks that separate the toes: They have separate toe sleeves that work with the natural shape of your foot and your foot's anatomy",
+    "copy-from": "socks"
+  },
+  {
+    "id": "splitsocks",
+    "type": "ARMOR",
+    "name": { "str": "pair of split toe socks", "str_pl": "pairs of split toe socks" },
```

Comment:
Any reason in particular why these aren’t just variants of the normal toe socks? There’s no difference in statistics, as far as I can see?

Reply:
I just realised you put this on draft, so I’m wagering you’re planning to add some form of difference, whatever that might look like. If not, though, the first comment applies, and if you are, apologies for being hasty with the review.

Reply:
Greetings Doctor, it is an honor to meet you once again. Now that you mentioned it, I don't see a reason why the split-toe socks are not variants of the same item. I'll fix it right now.